WORKSPACE

```diff
@@ -48,9 +48,9 @@ gazelle_dependencies()
 ### Standard python rules
 http_archive(
     name = "rules_python",
-    sha256 = "a3a6e99f497be089f81ec082882e40246bfd435f52f4e82f37e89449b04573f6",
-    strip_prefix = "rules_python-0.10.2",
-    url = "https://github.com/bazelbuild/rules_python/archive/refs/tags/0.10.2.tar.gz",
+    sha256 = "497ca47374f48c8b067d786b512ac10a276211810f4a580178ee9b9ad139323a",
```

Comment:
newer version required to allow compiling requirements.in for two different python versions

Reply:
Talk about a fractal...